react/src/components/calculators/late-interest/LateInterestContext.tsx

```diff
@@ -104,10 +105,10 @@ export function useLateInterestForm() {
   }, [InterestOnLateClassOneCalculator])
 
   useEffect(() => {
-    const taxYearData = buildTaxYears(ClassOneCalculator.getTaxYears)
+    const taxYearData = buildTaxYears(DirectorsCalculator.getTaxYearsWithOptions)

```

Comment:
Here I have used DirectorsCalc instead of ClassOne. Both come from NiFrontendInterface, which in turn uses the config from the config file. As Class One is set up to allow split years but Directors always deals in whole years, it was better to have these calcs build their tax year drop-down option from the director;s config